runtime/etc/lighttpd/lighttpd.conf

```diff
@@ -7,12 +7,4 @@ server.errorlog = "/dev/fd/2"
 accesslog.format = env.LIGHTTPD_ACCESSLOG_FORMAT
 accesslog.filename = "/dev/fd/2"
 
-fastcgi.server = ( "" =>
-  (
-	(
-	  "host" => env.LIGHTTPD_FASTCGI_HOST,
-	  "port" => env.LIGHTTPD_FASTCGI_PORT,
-	  "check-local" => "disable",
-	),
-  )
-)
+include_shell "bash ./gen-server"
```

Comment:
`include_shell` runs the script in a shell, so you are effectively having lighttpd run `sh -c "bash ./gen-server"` which is somewhat redundant.

Also, `./gen-server` is not very portable.  It will not work if you run lighttpd from a different directory, like testing in the root directory: `cd /; lighttpd -f /etc/lighttpd/lighttpd.conf -tt`